cypress/integration/functional/transaction/list.spec.ts

```diff
@@ -0,0 +1,29 @@
+context('app/transactions/list', () => {
+  describe('wallet has 2 transactions in history', function () {
+    before(() => {
+      cy.createEmptyWallet()
+
+      // setup transactions history, topup twice
+      const baseURL = '/v1/playground/rpc'
+      cy.intercept(`${baseURL}/sendtoaddress`).as('sendToAddress')
+      cy.getByTestID('playground_wallet_top_up').click()
+      cy.wait(['@sendToAddress'])
+      cy.getByTestID('playground_wallet_top_up').click()
+      cy.wait(['@sendToAddress'])
+
+      // balances to be mined
+      cy.wait(4000)
+
+      // go to page for testing
+      cy.getByTestID('bottom_tab_transactions').click()
+    })
+
+    it('should display 2 rows', () => {
+      cy.getByTestID('transactions_screen_list').should('exist')
+      cy.getByTestID('transaction_row_0').should('exist')
+      cy.getByTestID('transaction_row_1').should('exist')
+      // seems to be crashing (when trying to get element if testid not found)
+      // cy.getByTestID('transaction_row_2').should('not.exist')
```

Comment:
@thedoublejay this is what I mean by finding "expect to be not exist" element test fail, and it will cause the test time to be extreme long before it fail. I will disable it first until we can investigate further.

Reply:
Ok will check also. Syntax wise looks ok, maybe the usage of custom command breaks it but to be checked